docker/erlang.config

```diff
@@ -0,0 +1,76 @@
+%% -*- mode: erlang -*-
+[
+
+ {exometer, [{predefined, [
+    {[erlang, memory], {function, erlang, memory, [], value, []}, []},
+    {[erlang, system_info], {function, erlang, system_info, ['$dp'], value, [process_count]}, []},
+    {[erlang, statistics], {function, erlang, statistics, ['$dp'], value, [run_queue]}, []},
+    {[erlang, io], {function, erlang, statistics, [io], match, {{'_', input}, {'_', output}}}, []}
+   ]}
+ ]},
+
+ {mnesia, [
+    {dir, "/tmp/mnesia"}
+ ]},
```

Comment:
Is the mnesia dir the only change from erlang.config.in? We may then want to `sed` for this file too.